modules/logging-input-spec-filter-audit-infrastructure.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-input-spec-filtering.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-input-spec-filter-audit-infrastructure_{context}"]
+= Filtering application logs by filtering audit and infrastructure inputs by source
+
+You can define the list of `aduit` and `infrastructure` sources to collect application logs by using the `input` selector.
```

Comment:
These filtered logs are still `audit` and `infrastructure` logs, may I know why do you mention `application logs` here?

Reply:
I am removing "application logs" and changing it to only "logs". Hope it makes sense then. :)